scripts/build/termux_create_debian_subpackages.sh

```diff
@@ -103,7 +103,7 @@ termux_create_debian_subpackages() {
 
 		# If the subpackage is not in the $TERMUX_PKG_DEPENDS for the parent package,
 		# and TERMUX_SUBPKG_DEPEND_ON_PARENT doesn't have a value, the subpackage should depend on its parent
-		[[ " ${TERMUX_PKG_DEPENDS//,/ } " == *" $SUB_PKG_NAME "* ]] && : "${TERMUX_SUBPKG_DEPEND_ON_PARENT:=true}"
+		[[ " ${TERMUX_PKG_DEPENDS//,/ } " != *" $SUB_PKG_NAME "* ]] && : "${TERMUX_SUBPKG_DEPEND_ON_PARENT:=true}"
```

Comment:
Ah mixed up the condition did I?
Sorry, should have caught that in the initial PR.

Reply:
Easy to mix things like this up.

We should probably start building up a test suite - perhaps a subdirectory of minimal quick-building toy packages exercising various constructions, where we build each package there and verify that the built packages looks like expected in CI (on every change to build scripts). I'll create a draft to get things started in a future PR!